doc_source/mesh-k8s-integration.md

```diff
@@ -58,46 +58,38 @@ Install the integration components one time to each cluster that hosts pods that
 **Note**  
 The command creates an AWS IAM role with an auto\-generated name\. You are not able to specify the IAM role name that is created\.
 
    ```
    eksctl create iamserviceaccount \
        --cluster $CLUSTER_NAME \
        --namespace appmesh-system \
-       --name appmesh-controller \
+       --name appmesh-manager \
        --attach-policy-arn  arn:aws:iam::aws:policy/AWSCloudMapFullAccess,arn:aws:iam::aws:policy/AWSAppMeshFullAccess \
        --override-existing-serviceaccounts \
        --approve
    ```
 
    If you prefer to create the service account using the AWS Management Console or AWS CLI, see [Creating an IAM Role and Policy for your Service Account](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html#create-service-account-iam-role)\. If you use the AWS Management Console or AWS CLI to create the account, you also need to map the role to a Kubernetes service account\. For more information, see [Specifying an IAM Role for your Service Account](https://docs.aws.amazon.com/eks/latest/userguide/specify-service-account-role.html)\. 
 
-1. Deploy the App Mesh controller\. For a list of all configuration options, see [Configuration](https://github.com/aws/eks-charts/blob/master/stable/appmesh-controller/README.md#configuration) on GitHub\.
+1. Deploy the App Mesh controller manager\. For a list of all configuration options, see [Configuration](https://github.com/aws/eks-charts/blob/master/stable/appmesh-manager/README.md#configuration) on GitHub\.
 
    ```
-   helm upgrade -i appmesh-controller eks/appmesh-controller \
-       --namespace appmesh-system \
-       --set region=$AWS_REGION \
-       --set serviceAccount.create=false \
-       --set serviceAccount.name=appmesh-controller
+   helm upgrade -i appmesh-manager eks/appmesh-manager \
+    --namespace appmesh-system \
+    --set region=$AWS_REGION \
+    --set serviceAccount.create=false \
+    --set serviceAccount.name=appmesh-manager
    ```
 
-1. Confirm that the controller version is `v0.3.0` or later\. You can view the [change log](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md) on GitHub\.
+1. Confirm that the controller version is `v1.0.0` or later\. You can view the [change log](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md) on GitHub\.
 
    ```
-   kubectl get deployment -n appmesh-system appmesh-controller -o json  | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'
+   kubectl get deployment -n appmesh-system appmesh-manager -o json  | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'
```

Comment:
Deployment name should be "appmesh-controller-manager"

Reply:
Actually it is appmesh-manager

https://github.com/aws/eks-charts/tree/master/stable/appmesh-manager